src/Interpreters/InterpreterDropQuery.cpp

```diff
@@ -14,9 +14,13 @@
 #include <Common/escapeForFileName.h>
 #include <Common/quoteString.h>
 #include <Common/typeid_cast.h>
```

Comment:
include sorting is not correct. should reformat use clang-format

Reply:
This order is consistent with the original clickhosue, in order to prevent conflicts in future

Reply:
Let's avoid sorting existing source file which will have big impact for rebasing